core_lib/src/soundplayer.cpp

```diff
@@ -41,6 +42,10 @@ void SoundPlayer::init(SoundClip* clip)
 
     QFile file(clip->fileName());
     file.open(QIODevice::ReadOnly);
+    ScopeGuard fileScope([&] {
+        file.close();
```

Comment:
Same, this `file.close()` is not really necessary. If really want to close it explicitly, you can do it after `file.readAll()`

Reply:
Does it matter? The good thing about the ScoepGuard is that it happens at the end of the scope, so as long as we've initiated the ScopeGuard, it'll happen at the end no matter what. Right?

Reply:
Ah right, in that case i'll remove it again